src/shortcuts/lettercase.ts

```diff
@@ -0,0 +1,27 @@
+import Shortcut from '../@types/Shortcut'
+import Icon from '../components/icons/LetterCaseWithPicker'
+import isDocumentEditable from '../util/isDocumentEditable'
+
+/** Toggle the built-in =done style to cross out an item. */
+const lettercase: Shortcut = {
+  id: 'lettercase',
+  label: 'Lettercase',
+  description: 'Change the Letter case.',
+  svg: Icon,
+  canExecute: getState => isDocumentEditable() && !!getState().cursor,
+  multicursor: {
+    enabled: false,
+    error: () => 'Cannot change text color with multiple thoughts.',
+  },

```

Comment:
This shortcut should have multicursor support. Start with `multicursor: true` and see if that works. If not, we can discuss other multicursor strategies.